wagtail/admin/views/pages/revisions.py

```diff
@@ -16,26 +15,8 @@
 from wagtail.core.models import Page, UserPagePermissionsProxy
 
 
-@user_passes_test(user_has_any_page_permission)
 def revisions_index(request, page_id):
-    page = get_object_or_404(Page, id=page_id).specific
-
-    # Get page ordering
-    ordering = request.GET.get('ordering', '-created_at')
-    if ordering not in ['created_at', '-created_at', ]:
-        ordering = '-created_at'
-
-    revisions = page.revisions.order_by(ordering)
-
-    paginator = Paginator(revisions, per_page=20)
-    revisions = paginator.get_page(request.GET.get('p'))
-
-    return TemplateResponse(request, 'wagtailadmin/pages/revisions/index.html', {
```

Comment:
The revisions index template is still there.

Reply:
Good catch! Updated to remove that one now.